### PR TITLE
Remove activesupport dependency

### PR DESCRIPTION
**Pull request by srabuini**
What do you think about adding a simple helper and removing more than 11.000 lines of code?

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
that sounds great!!!!

**Comment by NullVoxPopuli** [comment]
can you add tests for the `present` method for things like `''`, `[]`, `{}`, `'not empty'`, `[0]`, `0`, etc?

**Comment by srabuini** [comment]
I've added *almost* the same blank/non_blank cases that activesupport tests.

Have a look, please, and let me know.

Thanks!

**Comment by NullVoxPopuli** [comment]
looking good! just one more change, I think!

File: spec/unit/message_spec.rb [hunk 1]
```diff
@@ -14,21 +14,48 @@
...
-    expected = expected.to_json
-
-    expect(msg.to_json).to eq expected
+  describe '#to_json' do
+    context 'when data is present' do
+      non_blank_data = [Object.new, true, 0, 1, 'a', [nil], { nil => 0 },
+                        Time.now]
+
+      non_blank_data.each do |data|
+        it "double the identifier and the data if the value would be #{data}" do
```
Comment (NullVoxPopuli) [inline spec/unit/message_spec.rb]: should inspect be used here as well?

Comment (srabuini) [reply spec/unit/message_spec.rb]: Yes. It produces almost the same output, just `a` vs `"a"` but should improve readability, I guess.

**Comment by NullVoxPopuli** [comment]
looks good! thanks! I'll release a new version shortly

**Comment by NullVoxPopuli** [comment]
@srabuini 2.0.1 released. :-)
Apparently I accidentally released 3. oops. that's been yanked.
